LICENSE.md

```diff
@@ -0,0 +1,27 @@
+# Disclaimer
+
+This license applies only to the source code that is directly maintained in
+this git repository, it does not extend to dependencies from outside of this
+repository, to include other projects owned and/or maintained by Kasm Technologies.
+
+# MIT License
+
+Copyright (c) 2022 Contributors of geocml-base-image
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
```

Comment:
I have a couple of conditions that I would like to add to the license here...

1. Derivative works must be licensed under this same license.
2. Changes made to derivative works must by merged to the upstream repo at github.com/geocml/geocml-base-image.

Not too hung up on the exact wording :smile:

Reply:
This is verbiage directly pulled from the MIT license. If we want to impose derivative works be licensed under the same license, we should consider using something like a GPL license

Reply:
Ahh crap, you're right--I can't get my licenses straight, I thought that MIT was the one that often had a weak copyleft clause attached to it (maybe I'm thinking of the BSD 2.0 license?). Anyway, I think that maybe we should consider GPL 2.0. I know that license has an upstream clause written into it, as well as built in copyleft protection. Is that something you're okay with?

Reply:
I believe BSD 2.0 is a permissive license that even has less restrictions than MIT. Nonetheless, I could get behind GPL v2 for this project.

Reply:
@capsulecorplab Per our offline conversation related to this, would you mind updating this PR to include the GPL v2 license?

Reply:
updated https://github.com/geoCML/geocml-base-image/pull/36/commits/35e71826fdd5a005c1e511f0a93f4405e228f1d7

Reply:
FYI, the last paragraph includes a clause that GPL v2 doesn't allow for linking as a library in proprietary products. Are we good with this or should we consider LGPL?

"This General Public License does not permit incorporating your program into proprietary programs. If your program is a subroutine library, you may consider it more useful to permit linking proprietary applications with the library. If this is what you want to do, use the GNU Lesser General Public License instead of this License."

Reply:
Good catch, but I think we're all good!  I kind of think about geoCML base image as an "Ubuntu-based Linux distro for GIS" versus a library.  geoCML isn't something that you would incorporate into proprietary software, rather, you may want to incorporate proprietary software _into_ a geoCML system.